audit/entry_formatter.go

```diff
@@ -30,7 +30,7 @@ var (
 
 // NewEntryFormatter should be used to create an EntryFormatter.
 // Accepted options: WithPrefix.
-func NewEntryFormatter(config FormatterConfig, salter Salter, opt ...Option) (*EntryFormatter, error) {
+func NewEntryFormatter(config FormatterConfig, salter Salter, headersConfig HeaderAdjuster, opt ...Option) (*EntryFormatter, error) {
```

Comment:
I think that the **EntryFormatter** can operate even if the headersConfig is `nil`. It would just mean that the step of examining the request headers would be skipped. This is what happens when the LogTestMessage is called.